literature/durand_1992_insitu_1977/durand_1992_insitu_1977_p1_f1a.yaml

```diff
@@ -0,0 +1,66 @@
+curation:
+  process:
+  - date: 2022-02-18
+    name: Tobias Schmider
+    orcid: https://orcid.org/0000-0002-2911-2925
+    role: curator
+  version: 1
+electrochemical system:
+  electrodes:
+    configuration: three
+    reference electrode:
+      type: RHE
+    working electrode:
+      crystallographic orientation: '110'
+      geometric electrolyte contact area:
+        unit: cm-2
+      material: Pt
+      preparation procedure: Annealed in an H2-O2 flame.
```

Comment:
Comments such as these should always be written as a full sentence (I fixed it).